util/argo/audit_logger.go

```diff
@@ -35,10 +35,13 @@ type ObjectRef struct {
 }
 
 const (
-	EventReasonWorkflowRunning  = "WorkflowRunning"
-	EventReasonWorkflowSucceded = "WorkflowSucceeded"
-	EventReasonWorkflowFailed   = "WorkflowFailed"
-	EventReasonWorkflowTimedOut = "WorkflowTimedOut"
+	EventReasonWorkflowRunning       = "WorkflowRunning"
+	EventReasonWorkflowSucceeded     = "WorkflowSucceeded"
+	EventReasonWorkflowFailed        = "WorkflowFailed"
+	EventReasonWorkflowTimedOut      = "WorkflowTimedOut"
+	EventReasonWorkflowNodeSucceeded = "WorkflowNodeSucceeded"
```

Comment:
should we say "Node" or "Step"?

Reply:
`Node`

Reply:
you sure? I feel like many user would understand "step"? @jessesuen - thoughts?

Reply:
We've gone through this discussion before: https://github.com/argoproj/argo/pull/1904#discussion_r382211882

`step` can be ambiguous as it doesn't include `dags`. In my opinion, the preferable term would be "task", but that ship has sailed, unfortunately.